static/js/stream_popover.js

```diff
@@ -500,6 +512,35 @@ exports.register_topic_handlers = function () {
 
         e.stopPropagation();
     });
+
+    $('body').on('click', '.sidebar-popover-move-topic-messages', function (e) {
+        const topic_row = $(e.currentTarget);
+        const stream_id = topic_row.attr('data-stream-id');
+        const topic_name = topic_row.attr('data-topic-name');
+        const message = home_msg_list.all_messages()
+            .find(m => m.topic === topic_name && m.stream_id === +stream_id);
```

Comment:
This check doesn't seem reliable; we may not have any messages from the topic cached in the frontend.  I think the right fix is to fetch the latest message ID from `stream_topic_history.js`; that is guaranteed to have data given that this widget is visible at all.

@showell is there an existing function we can call, or does one need to be added to expose "any message ID from a given stream/topic appearing in the left sidebar"?

Reply:
I don't think sending a message id is the right thing here. Shouldn't it be handled better on the server side using old and new streams and topics?

Reply:
Well, for this UI, yes; but we'll also add this to "edit a single message" UI, and for that, the message ID is an important aspect of the meaning.  (Basically whenever `propagate_mode != all`)

Reply:
Oh! That's good then.